pkgs/by-name/po/polybar/package.nix

```diff
@@ -111,6 +111,7 @@ stdenv.mkDerivation (finalAttrs: {
     '';
     license = licenses.mit;
     maintainers = with maintainers; [ afldcr Br1ght0ne moni ckie ];
+    mainProgram = "polybar";
```

Comment:
Maybe letting `nix run nixpkgs#polybar` run `polybar-msg` would be more useful? I don't personally use polybar so needing more opinions

Reply:
My original reasoning was that a user would want to `nix run` the bar itself to try out a config.

On one hand, if a person is already using Polybar, they might have it installed (via `systemPackages` or `home.packages`) and already have `polybar-msg` (which is useless if no polybar is running).

On the other hand, if they have a different bar (xmobar, yabar, or a builtin bar, like in DWM or Qtile) and would like to try Polybar out, I think for them `polybar` being `mainProgram` would be more useful.

The only reason for `mainProgram = "polybar-msg"` that I see is if a person is running a systemd service for Polybar and doesn't have it in PATH.